app/catalog-tab/launch/route.js

```diff
@@ -15,6 +15,7 @@ export default Route.extend({
   settings:     service(),
   growl:        service(),
   intl:         service(),
+  router:       service(),
 
```

Comment:
2.2 branch launch catalog. Fixed by adding this line. master branch works well without this line. cc @westlywright